test-cases/longevity/longevity-multi-keyspaces.yaml

```diff
@@ -21,6 +21,7 @@ n_loaders: 10
 n_monitor_nodes: 1
 
 instance_type_db: 'i3.8xlarge'
+root_disk_size_db: 60 # twice as as default, cause each keyspace generate lots of logs
```

Comment:
would it be enough? so, for this test case only, isn't it better to go even bigger?

Reply:
it's running for 48h now, seems stable enough.

![image](https://user-images.githubusercontent.com/340979/201897976-043aeefe-9b2d-4a0d-b26a-56c72e91d132.png)

![image](https://user-images.githubusercontent.com/340979/201898048-006d0949-e412-4bf7-88cb-a2cab3b2504a.png)